src/features/auth/pages/login/Login.native.test.tsx

```diff
@@ -320,19 +270,102 @@ describe('<Login/>', () => {
 
     expect(analytics.logSignUp).toHaveBeenNthCalledWith(1, { from: 'Login' })
   })
+
+  describe('Login from offer favorite modal', () => {
+    const OFFER_ID = addFavoriteJsonResponseSnap.offer.id
+
+    it('should redirect to Offer page when signin is successful and user comes from offer', async () => {
```

Comment:
le describe décrit déjà une parti du cas, on peut aléger tout les noms des tests

```suggestion
    it('should redirect to the previous offer page when signin is successful', async () => {
```